src/test/java/de/retest/recheck/RecheckImplIT.java

```diff
@@ -30,7 +30,7 @@
 	static final String MESSAGE_START_PATTERN = "A detailed report will be created at '.*'";
 	static final String MESSAGE_START_REPLACE = "A detailed report will be created at '*'";
 
-	static final Filter METADATA_FILTER = Filters.parse( "import: metadata.filter" );
+	static final Filter METADATA_FILTER = Filters.parse( "import: volatile-metadata.filter" );
```

Comment:
As seen in #757, I changed that to exclude all metadata, so this can left as `metadata.filter` (optional).